docs/build-from-scratch.md

```diff
@@ -32,6 +34,12 @@ sudo make all
 
 To push your image to a registry, use `make push`.
 
+### Keeping the Heptio registry tag
+
+If you don't wish to push your image to your own registry (not changing `$REGISTRY`), but simply want to build it locally with the Heptio tag, *you need to modify the deployment file that are using.* This is either `examples/azure/00-ark-deployment.yaml` if you're using Azure, or `examples/common/10-deployment.yaml` for all other cloud providers.
+
+Because the existing deployment files reference `gcr.io/heptio-images/ark:latest`, and Kubernetes handles the `latest` tag with the `imagePullPolicy` of "Always", your Ark deployment will pull from the Heptio image registry instead of using the local image. In order to use your local image, you'll need to specify a `$VERSION` tag (consistent with the value in the `Makefile`). If your `$VERSION` is "v0.3.3", for example, you need to set `spec.template.spec.containers[*].image` in your deployment file to be `gcr.io/heptio0images/ark:v0.3.3`.
```

Comment:
I'd recommend changing the imagePullPolicy. Or they can use `kubectl set image` after it's been deployed.